assets/messages.yml

```diff
@@ -117,6 +144,21 @@ messages:
         *Thank you for your report!*
         Could you please include information about your server? Without it we may be unable to reproduce the issue.
 
+        %quick_links%
+      fillname: []
+  attach-launcher-log:
+    - project:
+        - mc
+        - mcl
+      name: Attach launcher log
+      shortcut: llog
```

Comment:
Why not simply log? There doesn't seem to be any message that currently uses log anyways.

Reply:
Though `log` could be used for the game log in the future

Reply:
The game log is included in "Attach crash report".